LumpNav/LumpNav.py

```diff
@@ -40,18 +40,71 @@ def __init__(self, parent = None):
     GuideletWidget.__init__(self, parent)
     
   def setup(self):
+    # Adds default configurations to Slicer.ini
+    self.addDefaultConfiguration()
+    
     GuideletWidget.setup(self)
-
+    
   def addLauncherWidgets(self):
     GuideletWidget.addLauncherWidgets(self)
 
+    # Configurations
+    self.configurations()
+    
+    # BreachWarning
+    self.breachWarningLight()  
+ 
+  # Adds default configurations to Slicer.ini
+  def addDefaultConfiguration(self):
+    settings = slicer.app.userSettings()    
+    if not settings.value(self.moduleName + '/Configurations/default/TipToSurfaceDistanceCrossHair'):

```

Comment:
I tried both beginGroup() and childGroups() and I am pretty sure they doesn't work. beginGroup() just appends a prefix to a group and childGroups returns a list of all key top-level groups (i.e. LumpNav, General, MainWindow). I am pretty sure I either have to use the allKeys() approach or the one you just suggested, in which case I would prefer adding a name value since it is faster to implement. I will do that!